### DISCUSSION TITLE
[SOLVED] Choose a logo

### DISCUSSION THREAD

**Discussion by pioardi** [poolifier/poolifier#109] [Show and tell]
I guess is time to choose a cool logo for this project.
I drafted this but I will leave this discussion open for suggestions and feedbacks

![image](https://user-images.githubusercontent.com/15161576/107515221-05740000-6bab-11eb-8799-3561149e00dc.png)

**Comment by pioardi** [comment]
@Shinigami92 @jerome-benoit

**Reply by Shinigami92** [reply]
WOW looks really cool!
Could you draft some more with color(s)? Using green and/or orange colors maybe?

**Reply by pioardi** [reply]
Yeah I'll do other proposals going on

**Reply by pioardi** [reply]
![image](https://user-images.githubusercontent.com/15161576/107614252-40794080-6c4a-11eb-87b9-fbfbba07c65c.png)


![image](https://user-images.githubusercontent.com/15161576/107614285-4ec75c80-6c4a-11eb-99fb-ef5e3ec0062d.png)

![image](https://user-images.githubusercontent.com/15161576/107614331-64d51d00-6c4a-11eb-884c-2ae0b01b3031.png)

**Reply by jerome-benoit** [reply]
I like the last one.

**Reply by Shinigami92** [reply]
I like 1. and 3.

**Reply by pioardi** [reply]
I will give a try to the 3 as soon as I can

**Reply by Shinigami92** [reply]
For me, the contrast in 3 is better than the other two

**Comment by pioardi** [comment]
@jerome-benoit @Shinigami92  adding this logo here https://www.freepik.com/premium-vector/bear-vector-polar-bear-beach-swimming-pool-cartoon-character-illustration_3373887.htm#page=1&query=pool%20logo&position=9
If we will have funds in the future, we can consider to buy this logo :)

**Reply by Shinigami92** [reply]
I understand the joke, but I don't like it :confused:  
I prefer the current logo

**Reply by pioardi** [reply]
@Shinigami92  which joke are you referring to ?

**Reply by Shinigami92** [reply]
"pool"-ifier

**Reply by pioardi** [reply]
Yes is so nice lol, anyway if poolifier will grow I will consider to convert it into an organization and put in on open collective.
